fre/pp/checkoutScript.py

```diff
@@ -34,14 +34,31 @@ def _checkoutTemplate(experiment, platform, target):
     # Set the name of the directory
     name = f"{experiment}__{platform}__{target}"
 
-    # Get name of local branch and check branch rquest
-    branch_names = subprocess.run(["git","branch"],capture_output=True, text=True)
-    local_branch_name = branch_names.stdout.split()[1]
-    if branch != 'main':
-        if branch != local_branch_name:
-            stop_report = ("Error in branch: local branch does not match branch input")
-            sys.exit(stop_report)
-            return 1
+    # branch and version parameters
+    default_tag = subprocess.run(["fre","--version"],capture_output=True, text=True).stdout.split()[2]
+    if branch != 'empty':   
+        if os.path.isdir(name): #scenario 4
+            os.chdir(name)
+            name_path_tag=subprocess.run(["git","describe","--tags"],capture_output=True, text=True).stdout
+            name_path_branch=subprocess.run(["git","branch"],capture_output=True, text=True).stdout.split()[1]
```

Comment:
I'm worried about the split()[1] here.. is it safe and will it always be the second member?

`git branch` gives a lot of output. e.g.

```
(docs) c2b:~/git/fre-cli%>git branch  
  204-improve-fre-cmor-run-functionality
  CMORPytestUpdates
  catalog-merge
  checkout_edits
* docs
  main
  update-docs

Reply:
I am implementing changes to generalize this. When working locally, I only ever saw * current_branch' as the top item. But for other scenarios, new push should fix that.